### PR TITLE
Doc: textual fixes

### PR DESCRIPTION
**Pull request by RomkeVdMeulen**
I read through the entire doc and made small alterations that I think improve the readability somewhat.

There were a few things I wanted to discuss before I changed them out of hand:

> Since having access to the BehaviorSubject would allow consumers to directly emit the `next` value, instead of a front-facing `state` property, being an Observable which connects to the BehaviorSubject, is exposed.

I was unable to parse this sentence, and I don't know enough about RxJS to guess what was meant.

> Whether you prefer TypeScript or pure JavaScript is up to you, but having a typed state, allows for easier refactoring and better autocompletion support.

I modified this sentence slightly, but I wonder whether the remark should be part of this documentation at all.

> You're not forced to use async/await but it's highly recommended to use it for better readability wherever you can

Similar to the above.

> This is done in your app's `main` file, specifically the `configure` method.

Perhaps this should link to [App Configuration and Startup](https://aurelia.io/docs/fundamentals/app-configuration-and-startup)?

### CODE REVIEW

**Comment by zewa666** [comment]
Im not at a PC until the end of the month and digesting the changes via Phone is pretty hard ;) so please lets wait a bit with this PR as I'd like to check it in detail. Thanks for the great work.

**Comment by RomkeVdMeulen** [comment]
No problem. Much of it is a matter of taste anyway :)

File: doc/article/en-US/aurelia-store-plugin.md [hunk 1]
```diff
@@ -5,33 +5,33 @@ author: Vildan Softic (http://github.com/zewa666)
 ---
 ## Introduction
 
-This article covers the Store plugin for Aurelia. It is built on top of two core features of [RxJS](http://reactivex.io/rxjs/), namely Observables and the BehaviorSubject. You're not forced to delve into the reactive universe, in fact, you'll barely notice it at the begin but certainly can benefit a lot when using it wisely.
+This article covers the Store plugin for Aurelia. It is built on top of two core features of [RxJS](http://reactivex.io/rxjs/), namely Observables and the BehaviorSubject. You're don't have to delve into the reactive universe - in fact, you'll barely notice it at the beginning. But you can certainly benefit from having some knowledge about RxJS in order to use it wisely.
```
Comment (zewa666) [inline doc/article/en-US/aurelia-store-plugin.md]: You're --> You

Comment (zewa666) [inline doc/article/en-US/aurelia-store-plugin.md]: You're -> you

**Comment by zewa666** [comment]
Ok so I've walked through the changes and they all look pretty good, except a small typo. Now if you could resolve the merge-conflicts we're definitely good with this one.

With regards to your additional questions:

> Since having access to the BehaviorSubject would allow consumers to directly emit the next value, instead of a front-facing state property, being an Observable which connects to the BehaviorSubject, is exposed.

What I wanted to say is that if the BS would be public, devs could simply inject the store, grab the state and say getValue, thus negating the whole purpose of the system. States should flow from a dedicated master component down to its children, and should not be accessed indirectly by children themselves. Compare to functional composition vs having functions access global scopes.

>  App Configuration and Startup?
Yep that link is definitely helpful.

And with regards to async await vs TS/JS I'd leave them in, order to make clear what the expected benefits with the Plugin are, when using TS.

**Pull request merged**